config/environment.js

```diff
@@ -24,6 +24,7 @@ const logLevel = process.env.LOG_LEVEL || 'info'
 const pollingDisabled = 'DISABLE_POLLING' in process.env
 
 const rolePermittedAnnotation = process.env.ROLE_PERMITTED_ANNOTATION || 'iam.amazonaws.com/permitted'
+const namingPermittedAnnotation = process.env.NAMING_PERMITTED_ANNOTATION || 'iam.amazonaws.com/naming'
```

Comment:
Perhaps we want another annotation than this here, but Im not really sure what.
Since its not really related to iam and we its something that can be applied to all backendtypes   
`externalsecrets.kubernetes-client.io/permitted-key-name:` 
@silasbw thoughts?

Reply:
sgtm

Reply:
much better name.